src/client/table.rs

```diff
@@ -62,44 +56,43 @@ impl std::fmt::Debug for ObTable {
     }
 }
 
+// TODO: impl Table for ObTable
 impl ObTable {
-    pub fn execute_payload<T: ObPayload, R: ObPayload>(
+    pub async fn execute_payload<T: ObPayload, R: ObPayload>(
         &self,
         payload: &mut T,
         result: &mut R,
     ) -> Result<()> {
-        self.rpc_proxy.execute(payload, result)?;
+        self.rpc_proxy.execute(payload, result).await?;
         Ok(())
     }
 
-    pub fn query(&self, table_name: &str) -> impl TableQuery {
-        ObTableQueryImpl::new(table_name, Arc::new(self.clone()))
+    pub fn query(&self, _table_name: &str) {
+        // TODO: return impl TableQuery
+        todo!()
+        // ObTableQueryImpl::new(table_name, Arc::new(self.clone()))
     }
 
     pub fn operation_timeout(&self) -> Duration {
         self.config.rpc_operation_timeout
     }
 
-    fn execute(
+    /// Execute a batch operation on a table
+    pub async fn execute_batch(
```

Comment:
I guess we should move this method to the obclient if all the other methods are removed from the ObTable.

Reply:
Only keep the methods which are used by ObTableClient now, ``execute_batch()`` is used in batch operation, ``execute()`` is used by query.